chapter_preliminaries/index_vn.md

```diff
@@ -13,15 +13,19 @@ All machine learning is concerned with extracting information from data.
 So we will begin by learning the practical skills for storing, manipulating, and preprocessing data.
 -->
 
-*dịch đoạn phía trên*
+Để bắt đầu với học sâu, ta sẽ cần nắm bắt một vài kỹ năng cơ bản.
+Tất cả những vấn đề về học máy đều có liên quan đến việc trích xuất thông tin từ dữ liệu.
+Vì vậy, chúng tôi sẽ bắt đầu bằng cách học các kỹ năng thực tế để lưu trữ, thao tác và xử lý dữ liệu.
```

Comment:
```suggestion
Vì vậy, chúng ta sẽ bắt đầu bằng cách học các kỹ năng thực tế về lưu trữ, chỉnh sửa và tiền xử lý dữ liệu.
```

Reply:
@ngcthuong từ "manipulation" đã được dịch là "thao tác", liên quan đến các phép toán với `ndarray`. "Chỉnh sửa" chỉ là một phần nhỏ trong thao tác.